randomize/plugin.py

```diff
@@ -19,8 +19,37 @@
 from nose.util import isclass, isgenerator, transplant_func, transplant_class
 import random
 import unittest
+from random import getrandbits
+from nose.tools import nottest
 
 log = logging.getLogger(__name__)
+CLASS_SPECIFIC_RANDOMIZE_TESTS_FIELD_NAME = 'randomize_tests_seed'
+
+
+@nottest
```

Comment:
Exactly. Unless I decorate randomize_tests with @nottest, whenever I try to test my project (the one using nose-randomize), nose will consider it a test method, resulting in N+1 tests, having 
```The randomize_tests decorator ... ok
``` appear in the output.